plugins/obs-filters/nvidia-greenscreen-filter.c

```diff
@@ -248,6 +260,10 @@ static void init_images_greenscreen(struct nv_greenscreen_data *filter)
 	}
 
 	/* 5. Create and allocate Alpha NvCVimage (fx dst). */
+	if (filter->A_dst_img) {
+		NvCVImage_Destroy(filter->A_dst_img);
+		filter->A_dst_img = NULL;
+	}
```

Comment:
`NvCVImage_Realloc` might be usefull for some of the destroy/alloc combinations

Reply:
I removed the destroy calls, and tested resize with debugger breakpoints at all 5 spots. I assume NvCVImage_InitFromD3D11Texture has a "realloc" effect.